.cursor-plugin/skills/hooks-workflow/scripts/verify-hooks.sh

```
@@ -0,0 +1,246 @@
+PYEOF
+)" || true
+
+  if [[ -z "$commands_raw" ]]; then
+    add_check "hook-commands" "WARN" "no hook command entries found in file"; return
+  fi
+
+  local hook_idx=1
+  while IFS= read -r line; do
+    [[ -z "$line" ]] && continue
+    local rest event command token check_id probe_detail resolved
+    rest="${line#*:}"; event="${rest%%:*}"; command="${rest#*:}"
```

Shellcheck (0.11.0)

[warning] event appears unused. Verify use (or export if used externally).

(SC2034)